src/pages/home/report/ReportActionItemCreated.js

```diff
@@ -52,12 +52,11 @@ const ReportActionItemCreated = (props) => {
             onClose={() => Report.navigateToConciergeChatAndDeleteReport(props.report.reportID)}
         >
             <View style={StyleUtils.getReportWelcomeContainerStyle(props.isSmallScreenWidth)}>
-                <View pointerEvents="none" style={StyleUtils.getReportWelcomeBackgroundImageViewStyle(props.isSmallScreenWidth)}>
```

Comment:
I think that was from when I had initially used an 	`Image` instead of `ImageBackground` and we didn't want it to be selectable. https://github.com/Expensify/App/pull/13647#pullrequestreview-1232638704